lib/rubocop/cop/rails/deprecated_active_model_errors_methods.rb

```diff
@@ -118,6 +118,13 @@ def on_send(node)
 
         private
 
+        def skip_autocorrect?(node)
+          receiver = node.receiver.receiver
+          !AUTOCORRECTABLE_METHODS.include?(node.method_name) || (
+            receiver&.send_type? && receiver&.method?(:details) && node.method?(:<<)
+          )
```

Comment:
That just my two cents. Can you tweak it?

```suggestion
          return true unless AUTOCORRECTABLE_METHODS.include?(node.method_name)
          return false unless (receiver = node.receiver.receiver)

          receiver.send_type? && receiver.method?(:details) && node.method?(:<<)
```

Reply:
Sure, done.